import-service/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "import-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "main": "handler.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "aws-sdk": "^2.792.0",
```

Comment:
`aws-sdk` already preinstalled in the AWS environment, thus it could be moved to `devDependencies` to reduce bundle size (aws-sdk adds ~9Mb)

Reply:
How can I do that? Reinstall with --save-dev key? Can you share your Discord? I have several questions.

Reply:
yep, `npm i -D aws-sdk`, or just edit `package.json` file